### PR TITLE
Update Games Toolkit page

### LINKED ISSUE #9668: Update the existing Games Toolkit page (Include adding links to new templates and removing old template)
Update the current [Casual Games Toolkit](https://docs.flutter.dev/resources/games-toolkit) page content including updating copy and links to new templates.

Fixed by #9780

### PR DESCRIPTION
**Pull request by atsansone**
Fixes #9668

image

image

### CODE REVIEW

File: src/_layouts/base.html [hunk 1]
```diff
@@ -58,6 +58,7 @@
     <link href="https://fonts.googleapis.com/css2?family=Google+Sans+Text:wght@400;500;700&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Google+Sans+Mono:wght@400;500;700&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" rel="stylesheet" />
```
Comment (parlough) [inline src/_layouts/base.html:61]: Is there a reason you have to use Material Symbols instead of Icons? I don't really recommend it until we migrate everything over since that will increase site load time on every page not insignificantly. The file this loads is almost 3MB currently, which is likely bigger than everything else combined.

If you can't use Material Icons, consider adding Symbols for just this page and use a reduced set with just the variants you need specified in the URL.

Comment (atsansone) [reply src/_layouts/base.html:61]: It was an experiment. Let me see what I can do.

Comment (parlough) [reply src/_layouts/base.html:61]: Sounds good, thanks! Let me know if you have any issues :)

File: src/resources/games-toolkit.md [hunk 2]
```diff
@@ -1,241 +1,382 @@
...
-about them in quite this way.
+Video games fall into two categories: hobbyist games and casual games.
+You may be familiar with both types of games,
+though you might not have thought about them in quite this way.
 
-_Video games_ include arcade-style games: shooters,
-racing games, and so on.
-Video games are generally action-based and use a
-continual input device, like a joystick.
+_Hobbyist games_ cover games for those who dedicate time to the playing of games
```
Comment (sfshaza2) [inline src/resources/games-toolkit.md:23]: This first sentence uses "games" 3 times. Can you rephrase? Also, is this
definition strictly true? It strikes me as only true for one category of gamer.

File: src/resources/games-toolkit.md [hunk 3]
```diff
@@ -1,241 +1,382 @@
...
+with complex interfaces, rules, and controllers.
+These players are also known as core, hardcore, or professional gamers.
+These include arcade-style games: shooters, racing games, and so on.
+These types of games could use the [Flame game engine][] built using Flutter.
```
Comment (sfshaza2) [inline src/resources/games-toolkit.md:27]: It seems to me, that Flame has nothing to do with being a hobbyist. It has everything to do with the speed of game play.

File: src/resources/games-toolkit.md [hunk 4]
```diff
@@ -1,241 +1,382 @@
...
-like `provider`, `google_mobile_ads`, `in_app_purchase`,
-`audioplayers`, `crashlytics`, and `games_services`.
-Feel free to change the code
-to use other packages, if you prefer.
-
-And, yes, the game template assumes that **you'll
-want to earn money for your mobile game**,
-so the ads and in-app purchase packages are already
-integrated with the template and ready to be configured.
+* The [Build your own game in Flutter][game-video] video that explains
```
Comment (sfshaza2) [inline src/resources/games-toolkit.md:51]: Also, is this a new video?

File: src/resources/games-toolkit.md [hunk 5]
```diff
@@ -1,241 +1,382 @@
...
+  how to use the basic template.
+* The source code for a [finished tic tac toe game][],
+  built using the [basic game template][basic-template].
+  This game has been released on [Android][and-tic-tac] and [iOS][ios-tic-tac].
```
Comment (sfshaza2) [inline src/resources/games-toolkit.md:55]:
```suggestion
  This game has been released on the [Google Play store][and-tic-tac] and
  the [Apple Store][ios-tic-tac].
```

File: src/resources/games-toolkit.md [hunk 6]
```diff
@@ -1,241 +1,382 @@
...
-[Ads]: https://ads.google.com/intl/en_us/home/flutter/#!/
-[check your region's eligibility]: https://www.google.com/intl/en/ads/coupons/terms/flutter/
-[Cloud, Firebase]: https://cloud.google.com/free
-[Games]: {{site.main-url}}/games
+{{ site.alert.important }}
+**Terms and conditions apply.**
+You must connect your Firebase and GCP accounts to use credits for
+Firebase services and verify your business email during sign up to earn
+an additional $100 on top of the normal $300 credit.
+For the Ads offer, [check your region's eligibility][].
```
Comment (sfshaza2) [inline src/resources/games-toolkit.md:78]: Indent the content of all notes by 2 spaces on each line.

File: src/resources/games-toolkit.md [hunk 7]
```diff
@@ -1,241 +1,382 @@
...
-[direct-discord]: https://discord.com/login?redirect_to=%2Fchannels%2F420324994703163402%2F964110538986651658
-[game-discord]: https://discord.com/invite/WY5NwwjBQz
-[game-template]: {{site.github}}/flutter/samples/tree/main/game_template
-[game-video]: {{site.youtube-site}}/watch?v=zGgeBNiRy-8&t=3s
-[game-template-readme]: {{site.github}}/flutter/samples/tree/main/game_template#readme
+## Advanced game development
 
+Perhaps you want to write a hobbyist video game?
+The game template still provides value for
+the advanced interface and structure of such games.
```
Comment (sfshaza2) [inline src/resources/games-toolkit.md:104]: Such as?

File: src/resources/games-toolkit.md [hunk 8]
```diff
@@ -1,241 +1,382 @@
...
-For video games, you might want to look into the
-[Flame package][flame-package],
-join the [Flame community on Discord][flame-discord],
-and play with the [Flame demo app on DartPad][flame-demo].
-You might also check out the [Flutter + Flame codelab][].
+For Google I/O 2022, both the Flutter team and Very Good Ventures created new
+games.
 
-[Flutter + Flame codelab]: {{site.codelabs}}/codelabs/flutter-flame-game
+* VGV created the [I/O Pinball game][pinball-game] using the Flame engine.
```
Comment (sfshaza2) [inline src/resources/games-toolkit.md:115]: Do we know, for sure, that this is up to date?

File: src/resources/games-toolkit.md [hunk 9]
```diff
@@ -1,241 +1,382 @@
...
-[Tomb Toad]: https://play.google.com/store/apps/details?id=com.crescentmoongames.tombtoad
-
 ## Other resources
 
-Once you feel ready to go beyond the games template,
-you might want to investigate other resources
-recommended by the community.
-Bolded items in the following table are used in the games template:
+Once you feel ready to go beyond these games templates,
+investigate other resources that our community recommended.
```
Comment (sfshaza2) [inline src/resources/games-toolkit.md:134]:
```suggestion
investigate other resources that our community recommends.
```

File: src/resources/games-toolkit.md [hunk 10]
```diff
@@ -1,241 +1,382 @@
...
+{{ site.alert.secondary }}
+{{pkg-icon}} Flutter package<br>
+&nbsp;&nbsp;Packages included in the templates in **bold**<br>
+{{api-icon}} API documentation<br>
+{{codelab}} Codelab <br>
+{{recipe-icon}} Cookbook recipe<br>
+{{tool-icon}} Desktop application<br>
+{{assets-icon}} Game assets<br>
+{{engine}} Game engine<br>
+{{doc-icon}} Guide<br>
```
Comment (sfshaza2) [inline src/resources/games-toolkit.md:154]: Indent each line by 2 spaces. Why not just use a bulleted list?

File: src/resources/games-toolkit.md [hunk 11]
```diff
@@ -1,241 +1,382 @@
...
+{{ site.alert.end }}
 
 <table class="table table-striped" markdown="1">
-<tr markdown="1">
-  <th>Feature
-  </th>
-  <th>Resources
-  </th>
+<tr>
```
Comment (sfshaza2) [inline src/resources/games-toolkit.md:158]: I assume you've staged this and it looks ok?

**Review by sfshaza2 — COMMENTED** [review]
Reviewed.

File: src/_layouts/base.html (hunk 1 above)
Comment (atsansone) [reply src/_layouts/base.html:61]: Changed from dynamic to static. Now it's 239Kb

image

File: src/resources/games-toolkit.md (hunk 8 above)
Comment (atsansone) [reply src/resources/games-toolkit.md:136]: As far as we know, yes. Zoey didn't strike it.

File: src/resources/games-toolkit.md (hunk 11 above)
Comment (atsansone) [reply src/resources/games-toolkit.md:177]: It's in my second screenshot in the summary. Zoey was very happy with this.

File: src/_layouts/base.html (hunk 1 above)
Comment (parlough) [reply src/_layouts/base.html]: Still quite large but since we'll move everything else soon enough, that seems fine to me. Thanks!

File: src/resources/games-toolkit.md (hunk 10 above)
Comment (atsansone) [reply src/resources/games-toolkit.md]: Done. It looked weird to have a bullet, then an icon next to it.

**Review by atsansone — COMMENTED** [review]
PTAL. Changes made.

**Comment by domesticmouse** [comment]
Staged version?

**Comment by atsansone** [comment]
Can't. No Docker.

**Comment by domesticmouse** [comment]
> Can't. No Docker.

@parlough what is required to get the hosted staged preview versions for `flutter/website`?

**Comment by parlough** [comment]
I'm working on making setting up everything without Docker easier, but until then automatic staging is enabled if you open your branch directly on `flutter/website`.

With the Cloud Build work we'll be able to get rid of that limitation soon.

**Review by domesticmouse — APPROVED** [review]

**Comment by atsansone** [comment]
@domesticmouse : The only thing holding back the Cloud build enhancement is that the build logs don't always get uploaded to GitHub. That's a problem and there's a bug filed to resolve it.
